sycl/test-e2e/AddressSanitizer/use-after-free/use-after-free-1.cpp

```diff
@@ -0,0 +1,27 @@
+// REQUIRES: linux, cpu
```

Comment:
filename -1.cpp -2.cpp -3.cpp isn't very illustrative

Reply:
Well, this is some tests' naming style I found in "llvm/compiler-rt/test/asan/TestCases".
But, okay, I'll try to rename some of files to make it more meaningful.